mapit_gb/management/commands/mapit_UK_add_slugs_to_local_authorities.py

```diff
@@ -0,0 +1,38 @@
+# This script is for a one off import of govuk slugs for local authorities
+
+import json
+from django.core.management.base import NoArgsCommand
+from mapit.models import Area, CodeType
+from django.utils.six.moves import urllib
+
+
+class Command(NoArgsCommand):
+    help = 'Assigns slugs from frontend to local authorities'
+
+    def handle_noargs(self, **options):
+        try:
+            code_type = CodeType.objects.get(code='govuk_slug')
+        except CodeType.DoesNotExist:
+            code_type = CodeType(
+                code='govuk_slug',
+                description='Slug for use by GOV.UK'
+            )
+            code_type.save()
+
+        url = "https://raw.githubusercontent.com/alphagov/frontend/master/lib/data/authorities.json"
```

Comment:
We talked about this while implementing and thought this might be step 1: "Import from frontend".  We'd then follow this up with steps 2: "make frontend use mapit slugs instead of authorities.json", and 3: "move authorities.json out of frontend and into mapit as a fixture, changing this importer in the process".  Was a bit worried about creating another list (even if the intention is that it becomes the canonical list) and someone has to remember to update it as well if (when?) any changes are made.

Happy to skip straight to the step 3 (with a copy instead of a move) and deal with the duplication later when we're finally ready to remove authorities.json post step 2.